src/PhotonBackground.cpp

```diff
@@ -25,9 +27,20 @@ TabularPhotonField::TabularPhotonField(std::string fieldName, bool isRedshiftDep
 }
 
 
-double TabularPhotonField::getPhotonDensity(double Ephoton, double z) const {
-	if (this->isRedshiftDependent) {
-		return interpolate2d(Ephoton, z, this->photonEnergies, this->redshifts, this->photonDensity);
+double TabularPhotonField::getPhotonDensity(double Ephoton, double z) const {	
+	if ((this->isRedshiftDependent)) {
+		// fix behaviour for future redshift. See issue #414
+		// with redshift < 0 the photon density is set to 0 in interpolate2d. 
+		// Therefore it is assumed that the photon density does not change from z = 0. This is only valid for small changes.
```

Comment:
[...] from values at z = 0. [...] for small changes in redshift.

Reply:
done